app/schemas/com.yacgroup.yacguide.database.AppDatabase/1.json

```diff
@@ -0,0 +1,516 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "8303ae24eca627fcaebc04dd60fd4fd1",
+    "entities": [
+      {
+        "tableName": "Country",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`name` TEXT NOT NULL, PRIMARY KEY(`name`))",
+        "fields": [
+          {
+            "fieldPath": "name",
+            "columnName": "name",
+            "affinity": "TEXT",
+            "notNull": true
+          }
+        ],
+        "primaryKey": {
+          "columnNames": [
+            "name"
+          ],
+          "autoGenerate": false
+        },
+        "indices": [],
+        "foreignKeys": []
+      },
+      {
+        "tableName": "Region",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER NOT NULL, `name` TEXT, `country` TEXT, PRIMARY KEY(`id`))",
```

Comment:
All the field types should correspond to the types in the according database classes, e.g. `'name' TEXT` should correspond to `name: String?` in the `Region` class definition. 
What if you change this definition here without changing the class definition? 
- In my eyes the definition should be only in one single place. Having a class definition and a schema definition of one and the same database table is quite odd.
- I think, it's good to have a versioned database but I don't see the need of this schema since the sql statements are created automatically from the class definitions.

Reply:
- The file is exported automatically during build. Hence, any user-change will be overridden.  
- In the doc they stated as good practice to have this file but we can also set the `exportSchema = false`.
- We cannot influence the content of this file. I think they have a reason why they do it like this.

Reply:
Ah, I see. Then please

- [ ] Add the schemas folder to the `.gitignore` file